cqf-fhir-utility/src/main/java/org/opencds/cqf/fhir/utility/visitor/r4/KnowledgeArtifactPackageVisitor.java

```diff
@@ -352,6 +352,8 @@ protected void expandValueSet(
                         "Terminology Server expansion failed for: " + valueSet.getId(), ex.getMessage());
             }
         }
+
+        return valueSet;
```

Comment:
Question: If we're going to change it from `void` > `ValueSet`, would it be better to have it expand a copy of the input `ValueSet` and return the copy? 

If the method returns something then I feel like it would cause confusion if the method mutated the input _and_ returned a output at the same time?

Reply:
I reverted the return type as it would require to make ValueSet clonal which is not.